_config.yml

```diff
@@ -1 +1,46 @@
-theme: jekyll-theme-architect
\ No newline at end of file
+# Site settings
+title: Adulting Is Hard
+email: adult@adultingishard.com
+description: > # this means to ignore newlines until "baseurl:"
+  Congratulations! You have unlocked "Adult" form!
+  You may now use the Freedom ability!
+  But be careful, Responsibility and Stress can
+  cause massive damage!
+
+baseurl: "" # the subpath of your site, e.g. /blog/
+url: "http://adultingishard.com" # the base hostname & protocol for your site
```

Comment:
this will likely be adultingishard.io once I finish setting it up, the domain is approved.

Reply:
Looks like .io is taken (by you?) How about adultingishard.fyi? 
![image](https://user-images.githubusercontent.com/1191047/30675777-6e73e704-9e50-11e7-8a24-572559f37a0b.png)